common/persistence/sql/sqlplugin/matching_task_queue.go

```diff
@@ -64,6 +64,25 @@ type (
 		DataEncoding  string
 	}
 
+	AddBuildIdToTaskQueueMapping struct {
```

Comment:
Hmmm... yeah...
My point was that you're adding to the mapping of build id to task queue.
But if it doesn't read right to you, I can change (in a followup PR because I have a couple stacked on top of this).

Reply:
like